app/v2/notifications/post_notifications.py

```diff
@@ -281,6 +275,14 @@ def process_letter_notification(*, letter_data, api_key, template, reply_to_text
                 kwargs={'filename': filename},
                 queue=QueueNames.ANTIVIRUS,
             )
+        elif (
+            api_key.service.research_mode and
+            current_app.config['NOTIFY_ENVIRONMENT'] not in ['staging', 'live']
```

Comment:
If you change this, you need to add 'production' to the list. But I am wondering why you made this change? I find it easier to read conditions that are written in the positive.